pom.xml

```
@@ -52,14 +52,98 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.mapstruct</groupId>
+            <artifactId>mapstruct</artifactId>
+            <version>1.5.5.Final</version>
```

```suggestion
            <version>${mapstruct.version}</version>
```

create properties and use it in other necessary places